docs/recipes/babelrc.md

```diff
@@ -57,16 +84,17 @@ Set `babelrc` to `true`. This will use your `.babelrc` and extend it with any ad
 }
 ```
```

Comment:
Here? This one doesn't inherit anything you mean lower in the "Extends" section?